rundeckapp/grails-spa/packages/ui-trellis/src/app/pages/job/upload/JobUploadPage.vue

```diff
@@ -0,0 +1,542 @@
+        const fileInput = document.getElementById(
+          "xmlBatch",
+        ) as HTMLInputElement;
+        if (fileInput) {
+          fileInput.value = "";
+        }
+      } catch (error: any) {
+        console.error("Error uploading jobs:", error);
+        if (error.response?.data?.message) {
+          this.setError(error.response.data.message);
+        } else {
+          this.setError("An error occurred while uploading the file");
+        }
+      } finally {
+        this.uploading = false;
+      }
+    },
+    bulkActionPerformed(info:any){
+      if (info.action === 'delete') {
+        this.jobs = this.jobs.filter(job => !info.jobs.find((j) => j.id === job.id))
+      }
+    },
+    setError(message: string) {
+      this.errors.push(message);
+    },
+    cancel() {
+      // Redirect to jobs list
+      window.location.assign(`${rundeckContext.rdBase}/project/${this.project}/jobs`)
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="container-fluid">
+    <!-- Display errors if any -->
+    <div v-if="errors && errors.length" class="row">
+      <div class="col-sm-12">
+        <div class="alert alert-danger">
+          <div v-for="(msg, index) in errors" :key="index">{{ msg }}</div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Display error jobs if any -->
+    <div v-if="errjobs && errjobs.length" class="row">
+      <div class="col-sm-12">
+        <div class="card">
+          <div class="card-header text-danger">
+            <i class="glyphicon glyphicon-warning-sign"></i>
+            {{ $t("jobUpload.results.error.definitionErrors") }}
+          </div>
+          <div class="card-content">
+            <ul
+              v-for="(job, index) in errjobs"
+              :key="index"
+              class="list-unstyled "
+            >
+              <li class="flow-h">
+
+                <span class="jobname">
+                #{{ job.index }}:
+                  <a v-if="job.id" :href="job.permalink">{{
+                    job.name || "(Name missing)"
+                  }}</a>
+                  <template v-else>{{ job.name }}</template>
+                </span>
+                <span class="jobdesc" style="">{{
+                  job.description && job.description.length > 100
+                    ? job.description.substring(0, 100)
+                    : job.description
+                }}</span>
+                <div class="errors">
+                  <template v-if="job.error">{{ job.error }}</template>
+                </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Display skipped jobs if any -->
+    <div v-if="skipjobs && skipjobs.length" class="row">
+      <div class="col-sm-12">
+        <div class="card">
+          <div class="card-header text-info">
+            <i class="glyphicon glyphicon-info-sign"></i>
+            {{ skipjobs.length }} {{ $tc("jobUpload.jobs", skipjobs.length)
```

Comment:
```suggestion
            {{ skipjobs.length }} {{ $t("jobUpload.jobs", skipjobs.length)
```

$tc will be dropped completely in vue i18n 11, it would be good to avoid using it as $t already supports the same logic